code/Example/Drasil/GlassBR/Body.hs

```diff
@@ -472,44 +472,44 @@ assumptions = fst (foldr (\s (ls, n) -> ((Assumption $ assump ("A" ++ show n) s
 
 {-input and output tables-}
 
-s6_2_5_table1 = inDataConstTbl gbInputDataConstraints
-s6_2_5_table2 = outDataConstTbl [prob_br]
+data_consts_table1 = inDataConstTbl gbInputDataConstraints
+data_consts_table2 = outDataConstTbl [prob_br]
 
-s6_2_5_intro2 :: Sentence
-s6_2_5_intro2 = foldlSent [makeRef s6_2_5_table2, S "shows the",
+data_consts_intro2 :: Sentence
+data_consts_intro2 = foldlSent [makeRef data_consts_table2, S "shows the",
   plural constraint, S "that must be satisfied by the", phrase output_]
 
 {--REQUIREMENTS--}
 
 {--Functional Requirements--}
 
-s7_1_list = (s7_1_listOfReqs) ++ s7_1_req6 ++ [s7_1_req1Table]
+func_reqs_list = (func_reqs_listOfReqs) ++ func_reqs_req6 ++ [func_reqs_req1Table]
 
-s7_1_req1, s7_1_req2, s7_1_req3, s7_1_req4, s7_1_req5 :: Contents
+func_reqs_req1, func_reqs_req2, func_reqs_req3, func_reqs_req4, func_reqs_req5 :: Contents
 req1Desc, req2Desc, req3Desc, req4Desc :: Sentence
 req5Desc :: NamedChunk -> Sentence
-s7_1_req6 :: [Contents] --FIXME: Issue #327
+func_reqs_req6 :: [Contents] --FIXME: Issue #327
 
-s7_1_listOfReqs :: [Contents]
-s7_1_listOfReqs = [s7_1_req1, s7_1_req2, s7_1_req3, s7_1_req4, s7_1_req5]
+func_reqs_listOfReqs :: [Contents]
+func_reqs_listOfReqs = [func_reqs_req1, func_reqs_req2, func_reqs_req3, func_reqs_req4, func_reqs_req5]
 
-s7_1_req1 = mkRequirement "s7_1_req1" req1Desc "Input-Glass-Props"
-s7_1_req2 = mkRequirement "s7_1_req2" req2Desc "System-Set-Values-Following-Assumptions"
-s7_1_req3 = mkRequirement "s7_1_req3" req3Desc "Check-Input-with-Data_Constraints"
-s7_1_req4 = mkRequirement "s7_1_req4" req4Desc "Output-Values-and-Known-Quantities"
-s7_1_req5 = mkRequirement "s7_1_req5" (req5Desc (output_)) "Check-Glass-Safety"
+func_reqs_req1 = mkRequirement "func_reqs_req1" req1Desc "Input-Glass-Props"
+func_reqs_req2 = mkRequirement "func_reqs_req2" req2Desc "System-Set-Values-Following-Assumptions"
+func_reqs_req3 = mkRequirement "func_reqs_req3" req3Desc "Check-Input-with-Data_Constraints"
+func_reqs_req4 = mkRequirement "func_reqs_req4" req4Desc "Output-Values-and-Known-Quantities"
+func_reqs_req5 = mkRequirement "func_reqs_req5" (req5Desc (output_)) "Check-Glass-Safety"
```

Comment:
Should the string be left unchanged, as it functions as the `refAdd`? I figured that the numbered sections should be replaced entirely, so I updated stable to reflect this change. @smiths